mkt/api/tests/__init__.py

```diff
@@ -19,7 +21,8 @@ def _allowed_verbs(self, url, allowed):
             if verb in allowed:
                 continue
             try:
-                res = getattr(self.client, verb)(url)
+                with transaction.atomic():
```

Comment:
This silenced a bunch of "no more db access allowed in this transaction" errors in the tests.